wolfcrypt/benchmark/benchmark.c

```diff
@@ -7295,7 +7305,137 @@ void bench_falconKeySign(byte level)
     wc_falcon_free(&key);
 }
 #endif /* HAVE_FALCON */
-#endif /* HAVE_PQC && HAVE_LIBOQS */
+
+#ifdef HAVE_DILITHIUM
+void bench_dilithiumKeySign(byte level, byte sym)
```

Comment:
Can you update the documentation with benchmarks for Dilithium?

Reply:
I have not done this in the latest update. 
Stay tuned.